code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -1,3 +1,5 @@
+#define SPEAK(message) radio.talk_into(src, message, radio_channel, get_spans())

```

Comment:
@Jordie0608 Looks like it needs to be a define, when i try and use "radio.talk_into(src, "message here") it just broadcasts to the station wide radio